docs/dev/reference/dca/fields.md

```diff
@@ -188,7 +188,8 @@ Each field can be validated against a regular expression.
 | unique             | true/false (`bool`)           | If true the field value cannot be saved if it exists already.                                                                                                            |
 | uploadFolder       | Path (`string`)                  | The target path for file uploads of the `upload` widget.                                                                                                              |
 | useRawRequestData  | true/false (`bool`)           | If true the raw request data from the Symfony request is used. **Warning:** input filtering is bypassed! Make sure the data is never output anywhere in the back end unescaped which it would if you added the field to a regular back end list view for example. |
-| versionize         | true/false (`bool`)           | If false skip this field in the versioning. Default `true`.
+| versionize         | true/false (`bool`)           | If false skip this field in the versioning. Default `true`. |
+| basicEntities | true/false (`bool`) | {{< version-tag "5.0" >}} If true conversts basic entities like `&shy;`, `&amp;` etc. back to their Contao representation `[&]`, `[-]` etc. |
```

Comment:
```suggestion
| basicEntities | true/false (`bool`) | {{< version-tag "5.0" >}} If true converts basic entities like `&shy;`, `&amp;` etc. back to their Contao representation `[&]`, `[-]` etc. when editing and vice versa when saving. |
```

Reply:
That wouldn't be entirely corect, as the vice versa part always happens, regardless of this setting, doesn't it?

Reply:
No, both conversions only happen if this setting is activated.